dask/multiprocessing.py

```diff
@@ -164,7 +167,14 @@ def get(
         Function to use for function deserialization (defaults to cloudpickle.loads)
     optimize_graph : bool
         If True [default], `fuse` is applied to the graph before computation.
+    pool : Executor or Pool
+        Some sort of `Executor` or `Pool` to use
+    chunksize: int, optional
+        Size of chunks to use when dispatching work.
+        Defaults to 5 as some batching is helpful.
+        If -1, will be computed to evenly divide ready work across workers.
     """
+    chunksize = chunksize or config.get("chunksize", 6)
```

Comment:
cc @jcrist (as this may also be relevant to Prefect as it sounds like the Multiprocessing Pool is used)

Opened issue ( https://github.com/dask/dask/issues/7510 ) about potentially refining this and documenting it better

Reply:
Thanks for flagging this @jakirkham -- it made it a lot easier to track down where this came from when users started encountering a "lack of parallelism" with trivial flows. I'll be maintaining the Dask executors in Prefect now and would appreciate similar pings in the future if you remember :)